WordPress/Classes/ViewRelated/Post/WPPostViewController.m

```diff
@@ -873,6 +921,23 @@ - (void)showMediaPickerWithDataSource:(id<WPMediaCollectionDataSource>)dataSourc
     [self presentViewController:picker animated:animated completion:nil];
 }
 
+- (void)showDocumentPickerAnimated:(BOOL)animated
+{
+    NSArray *docTypes = @[[NSString stringWithFormat:@"%@", kUTTypeImage],
+                          [NSString stringWithFormat:@"%@", kUTTypeMovie]];
+    UIDocumentPickerViewController *docPicker = [[UIDocumentPickerViewController alloc]
+                                                 initWithDocumentTypes:docTypes inMode:UIDocumentPickerModeImport];
+    docPicker.delegate = self;
+    
+    // The app's appearance settings override the doc picker color scheme.
+    // Setting the nav colors here so the doc picker has the correct appearance.
+    // The app colors are restored later with resetNavigationColors().
+    [[UINavigationBar appearance] setTintColor:[WPStyleGuide mediumBlue]];
```

Comment:
I think now we have two places where we set these colours (here and in the media library) we could probably extract these into a `WPStyleGuide` extension / method that we call from both places. Then, if ever we need to change this colour, we just have to do it in one place.

Check out our other WPStyleGuide extensions for an example :)